tools/src/dynamic-macros/macros.ts

```diff
@@ -198,7 +198,7 @@ export default {
       return '';
     }
 
-    const pathToHome = 'home';
+    const pathToHome = 'apps/expo-go';
     const url = await UrlUtils.constructManifestUrlAsync(path.join(EXPO_DIR, pathToHome));
```

Comment:
If you agree with introducing a new constant 👇 
```suggestion
    const url = await UrlUtils.constructManifestUrlAsync(EXPO_GO_DIR);
```